inlong-manager/manager-dao/src/main/resources/mappers/StreamSourceEntityMapper.xml

```diff
@@ -271,6 +271,18 @@
         </where>
     </select>
 
+    <update id="updateByGroupIdAndStreamId">
+        update stream_source
+        <set>
+            is_deleted = id,
+            previous_status = status,
+            status = #{status, jdbcType=INTEGER},
```

Comment:
It is recommended to modify the `version` to prevent the deleted source from being updated during concurrent modification.

Reply:
Done.